task/07-yield-tasks.js

```diff
@@ -33,10 +33,24 @@
  *
  */
 function* get99BottlesOfBeer() {
```

Comment:
The whole story with **line** variable and returning iterator variable  to prev state `i<99 ? i+=1 : i` is makin no sense. if you need to do 2 things in one iteration. Just do them one after another in one iteration.